src/redux/explorer.js

```diff
@@ -422,26 +420,57 @@ export const explorerInitL2 = (network = null) => (dispatch, getState) => {
   if (getState().settings.network === NetworkTypes.mainnet) {
     switch (network) {
       case NetworkTypes.arbitrum:
-        // Start watching arbitrum assets
-        dispatch(arbitrumExplorerInit());
+      case NetworkTypes.polygon:
+        // Start watching arbitrum and polygon assets
+        dispatch(fetchAssetsFromRefraction());
         break;
       case NetworkTypes.optimism:
         // Start watching optimism assets
         dispatch(optimismExplorerInit());
         break;
-      case NetworkTypes.polygon:
-        // Start watching polygon assets
-        dispatch(polygonExplorerInit());
-        break;
       default:
         // Start watching all L2 assets
-        dispatch(arbitrumExplorerInit());
         dispatch(optimismExplorerInit());
-        dispatch(polygonExplorerInit());
     }
   }
 };
 
+const fetchAssetsFromRefraction = () => (dispatch, getState) => {
```

Comment:
do we actually need this? is refraction only sending us the response on `get` instead of `subscribe`? Otherwise, this is redundant no?

Reply:
Yes, this is a new behavior wayne introduced to "pull" on demand because it will hit addys with L2, which otherwise gets only updated on an interval